### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by Gnome-py**
Reword .emureco
Reword .stock 3ds slightly
Add .luma updater to replace .tinysearch updater
Remove .ctrmount (was used to help last in 2017)
Alias .7z to .p7zip because I can never remember if that is a valid alias

### CODE REVIEW

File: cogs/assistance.py [hunk 1]
```diff
@@ -405,7 +405,7 @@ async def stock(self, ctx, console=None):
         if self.check_console(console, ctx.message.channel.name, '3ds'):
             embed = discord.Embed(title="Running stock (unmodified) 11.4+ firmware?", color=discord.Color.dark_orange())
             embed.add_field(name="NTRBoot", value="Requires a compatible NDS flashcart and maybe an additional DS(i) or hacked 3DS console depending on the flashcart (All versions, all hardware). [Guide](https://3ds.hacks.guide/ntrboot)", inline=False)
-            embed.add_field(name="Seedminer", value="Requires a working NDS mode or Pokémon Picross (free from eshop) [Guide](https://3ds.hacks.guide/seedminer)", inline=False)
+            embed.add_field(name="Seedminer", value="Requires a working DSiWare Data Management, NDS mode, or Pokémon Picross (free from eshop) [Guide](https://3ds.hacks.guide/seedminer)", inline=False)
```
Comment (thedax) [inline cogs/assistance.py:408]: Isn't this already implied by "working NDS mode"?

File: cogs/assistance.py [hunk 2]
```diff
@@ -721,7 +721,7 @@ async def emureco(self, ctx, console=None):
         if self.check_console(console, ctx.message.channel.name, '3ds'):
             embed = discord.Embed(title="EmuNAND for 3DS", color=0xe60012)
             embed.description = cleandoc("""
-            With the recent advances in hacking methods and safety, it is no longer recommended to use an emuNAND on a 3DS/2DS system.
+            Since 2016 it has no longer been recommended to use an emuNAND on a 3DS/2DS system.
```
Comment (thedax) [inline cogs/assistance.py:724]: "As of 2016, it is no longer recommended to use an emuNAND on a 3DS/2DS system."

File: cogs/assistance.py [hunk 3]
```diff
@@ -771,6 +763,10 @@ async def luma(self, ctx, lumaversion=""):
             await self.simple_embed(ctx, f"Luma v{lumaversion}\nhttps://github.com/LumaTeam/Luma3DS/releases/tag/v{lumaversion}", color=discord.Color.blue())
         elif lumaversion == "latest":
             await self.simple_embed(ctx, "Latest Luma Version:\nhttps://github.com/LumaTeam/Luma3DS/releases/latest", color=discord.Color.blue())
+        elif lumaversion == "updater":
```
Comment (thedax) [inline cogs/assistance.py:766]: This isn't desired at the current time.

**Comment by FrozenChen** [comment]
Since the review hasnt been resolved for months , i will close this PR

**Pull request closed without merging**